documentation/src/main/asciidoc/topics/ref_ports_protocols.adoc

```diff
@@ -0,0 +1,52 @@
+[id='ports_protocols-{context}']
+= Ports and Protocols
+{brandname} uses the following ports and protocols to allow network traffic
+between remote endpoints and clustered nodes:
+
+[%header,cols=3*]
+|===
+|Port
+|Protocol
+|Description
+
+|11222
+|TCP
+|Hot Rod and REST endpoint access
+
+|11221
```

Comment:
@tristantarrant do you remember how we ended up using 11221 as the default memcached port? In the WildFly-based server it used to be 11211 (Memcached's default):
https://github.com/infinispan/infinispan/blob/9.4.x/server/integration/endpoint/src/main/resources/subsystem-templates/infinispan-endpoint.xml#L166